rasa/core/actions/forms.py

```diff
@@ -605,9 +605,21 @@ async def activate(
 
         if not prefilled_slots:
             logger.debug("No pre-filled required slots to validate.")
+        else:
+            logger.debug(f"Validating pre-filled required slots: {prefilled_slots}.")
+
+        validate_name = f"validate_{self.name()}"
+
+        if validate_name not in domain.action_names_or_texts:
+            logger.debug(
+                f"There is no validation action '{validate_name}' "
+                f"to execute at form activation."
+            )
             return [e for e in extraction_events if isinstance(e, SlotSet)]
```

Comment:
We should strive away from using single letter as name for a variable.
Consider using `extraction_event` instead of `e`.

Reply:
Yep, legacy naming code that I forgot to update in this PR, I'll update 👍🏻